tests/test_inputs.py

```diff
@@ -867,7 +831,28 @@ def check_cells_equal(actual_cells, expected_cells) -> None:
 
     for index in chain(range(3, 6), range(7, 21)):
         field = f"f{index}"
-        assert getattr(actual_cells, field) == getattr(expected_cells, field)
+        if index != 14:
+            assert getattr(actual_cells, field) == getattr(expected_cells, field)
+        else:
```

Comment:
You should change his to put the test for f14 outside of the loop (as is done for f6 above). Change the chain to `chain(range(3, 6), range(7, 14), range(15, 21))` and move the code following the else out of the loop.